models/dbt_queries.sql

```diff
@@ -0,0 +1,25 @@
+select
+    *,
```

Comment:
```suggestion
    * except dbt_metadata,
```

do we still want to include that field?